ddtrace/contrib/celery/patch.py

```diff
@@ -14,13 +16,18 @@ def patch():
     case of Django-Celery integration, also the `@shared_task` decorator
     must be instrumented because Django doesn't use the Celery registry.
     """
+    # instrument the main Celery application constructor
     setattr(celery, 'Celery', patch_app(celery.Celery))
+    # `app.Task` is a `cached_property` so we need to patch the base class
+    # that is used to create this one.
+    patch_task(celery.app.task.Task)
```

Comment:
`setattr(app, 'Task', patch_task(app.Task))` in the other module, has been changed with this statement.